aea/connections/scaffold/connection.py

```diff
@@ -33,22 +31,15 @@ class MyScaffoldConnection(Connection):
 
     connection_id = PublicId.from_str("fetchai/scaffold:0.1.0")
 
-    def __init__(
-        self,
-        configuration: ConnectionConfig,
-        identity: Optional[Identity] = None,
-        crypto_store: Optional[CryptoStore] = None,
-    ):
+    def __init__(self, **kwargs):
         """
         Initialize a connection to an SDK or API.
 
         :param configuration: the connection configuration.
         :param crypto_store: object to access the connection crypto objects.
         :param identity: the identity object.
```

Comment:
The docstring is no longer accurate.

Reply:
updated to the base connection definition